cmds/report.c

```diff
@@ -569,6 +590,9 @@ static void report_diff(struct uftrace_data *handle, struct opts *opts)
 	};
 	int h_idx = (avg_mode == AVG_NONE) ? 0 : (avg_mode == AVG_TOTAL) ? 1 : 2;
 	int f_idx = diff_policy.percent ? 1 : (avg_mode == AVG_NONE) ? 0 : 2;
+	char field_func_name[] = "Function [Source]";
+	if (!opts->srcline)
```

Comment:
Please give an empty line after declaration statements.

Reply:
ok.

Reply:
It's not updated yet.